src/cmd/placement/open.js

```diff
@@ -4,14 +4,12 @@ const getPkg = require('../../lib/get-pkg')
 const log = require('../../lib/log')
 const opn = require('opn')
 
-module.exports = async function open (page = 'editor', options) {
+module.exports = async function open (page = 'editor', options = {}) {
   await throwIf.placement('open')
   const pkg = await getPkg()
   const { propertyId, placementId } = pkg.meta
 
-  const url = `${
-    config.services.app
-  }/p/${propertyId}/atom/placements/${placementId}/create?step=editor&view=schema`
+  const url = `${config.services.app}/p/${propertyId}/atom/placements/${placementId}/create?step=${page}&view=schema`
   log.info(`opening ${url}`)
-  return opn(url)
+  return opn(url, { ...options, wait: false })
```

Comment:
don't pass options in here plz

Reply:
those options are command line options and are likely to be different. e.g. we could have options to open different urls